code/search/fibonacci_search/fibonacci_search.js

```diff
@@ -0,0 +1,52 @@
+function fibonacciSearch(ar,x,n) {
+	var secondPreceding = 0,
+	firstPreceding = 1;
```

Comment:
Any reason why you aren’t using var here?

Reply:
There is a comma to separate the two.

Reply:
Oh right, didn’t see that. I think you should put both variables on the same line.